### ISSUE TITLE
Complete pyg4ometry docker image

### ISSUE THREAD

**Issue by stewartboogert** [g4edge/pyg4ometry#78]
We should have a complete docker image to run pyg4ometry (opposed to the build environment for testing) 

- I think it would be nice if changes in the docker file triggered a container build on docker hub. Do we already do this? 
- There is lots of instruction on getting the graphics running (in bitbucket issues) within a container we can consolidate in the manual
- I think we should pay for  more significant docker resources

**Comment by gipert** [comment]
I *should* have enabled automated builds on DockerHub, triggered by pushes and PRs: https://hub.docker.com/repository/docker/g4edge/pyg4ometry/builds

PS: not sure about resources limits on DockerHub.

**Comment by stewartboogert** [comment]
Right, but where should we put the docker file that **includes** the build of pyg4ometry and VNC/Xwindows etc. I could put a directory like  ```ubuntu-pyg4ometry``` in the docker repo

**Comment by gipert** [comment]
I would propose to name the base Docker images just `ubuntu`, `manylinux2014_x86_64`, ... and create a new "production ready" image (with pyg4ometry) named just `pyg4ometry". I would install any extra dependencies like VNC/Xwindows in the base images.
